pebble/common/types.py

```diff
@@ -26,8 +26,9 @@ def __init__(self, msg, code=0):
         super(ProcessExpired, self).__init__(msg)
         self.exitcode = code
 
+_T = TypeVar("_T")
```

Comment:
Here you could do:
```python
try:
    from typing import TypeVar
    _T = TypeVar("_T")
except ImportError:
    _T = "GenericType"
```

Have not tested it but it might be enough of a change to support old interpreters.